{{cookiecutter.repostory_name}}/app/src/requirements.txt

```diff
@@ -4,6 +4,7 @@ django-csp==3.6
 {% endif -%}
 django-environ==0.4.5
 django-extensions==2.1.6
+django-probes==1.5.0
```

Comment:
what is this for?

Reply:
For `docker-compose exec -T app python manage.py wait_for_database`